kubernetes/data_source_kubernetes_endpoints_v1.go

```diff
@@ -0,0 +1,35 @@
+package kubernetes
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func dataSourceKubernetesEndpointV1() *schema.Resource {
```

Comment:
Could you please rename `dataSourceKubernetesEndpointV1` to `dataSourceKubernetesEndpointsV1`?